### ISSUE TITLE
Jigzi Player: "Find the place" - missing voiceover

### ISSUE THREAD

**Issue by chayaml** [ji-devs/ji-cloud#2454]
The voiceover is missing from this game as well. Seems like there is a recurrent issue?

https://jigzi.org/admin/curation/4fbc02ac-69c0-11ec-976d-17f5cc06a271

https://jitap.net/activities/gz7/play/

**Comment by corinnewo** [comment]
@dakom This is a series of questions on the same page - ASK A QUESTION. The whole game is only 1 page. If the voiceover is missing, can't do the interaction at all. Please look into this.

**Comment by dakom** [comment]
@corinnewo - I can't see https://jitap.net/activities/gz7/play/ - when I visit it I get redirected to https://jitap.net/pages/ji_tap_pro_plans_teacher/

**Comment by corinnewo** [comment]
@dakom please log in with my credentials - my email and TalNad - I need to organise you a gift account!!

**Comment by corinnewo** [comment]
Similar to #2453

**Comment by dakom** [comment]
duplicate of #2578
